config/routes.rb

```diff
@@ -55,6 +55,12 @@ def self.matches?(request)
     member do
       post 'duplicate_with_associations'
       post 'duplicate_without_associations'
+      get 'export_with_associations_as_json'
+      get 'export_without_associations_as_json'
+    end
+
+    collection do
```

Comment:
@alis-khadka what do you think about taking the route out of the collection block? I believe this should just be a stand-alone route like #new or #create